### DISCUSSION TITLE
Normalization

### DISCUSSION THREAD

**Discussion by okeneo** [dcaribou/transfermarkt-datasets#237] [Q&A] [answered]
Hello, does anyone know if this dataset is normalized?

**Comment by dcaribou** [comment, marked as answer]
Hi @okeneo 

If your question is regarding the database model (ie, normalised vs. de-normalised) then I would say that although it is mostly normalised no strict strategy is followed.

For example, for convenience, tables may contain redundant attributes (like the club name appearing in `clubs` and in games `tables`) or the same data may appear in multiple tables in a different format (see `games` and `club_games`) which would be against normalisation principles.

Convenience for data consumers is prioritised, rather than one modelling technique in particular.

**Reply by okeneo** [reply]
Thank you very much! Awesome dataset!!
